libvcx/src/libindy/utils/wallet.rs

```diff
@@ -375,7 +368,8 @@ pub mod tests {
 
         let export_file = TempFile::prepare_path(wallet_name);
 
-        let _handle = create_and_open_as_main_wallet(wallet_name, settings::DEFAULT_WALLET_KEY, settings::WALLET_KDF_RAW, None, None, None).unwrap();
+        let wallet_config = WalletConfig { wallet_name: wallet_name.into(), wallet_key: settings::DEFAULT_WALLET_KEY.into(), wallet_key_derivation: settings::WALLET_KDF_RAW.into(), wallet_type: None, storage_config: None, storage_credentials: None, rekey: None, rekey_derivation_method: None };
```

Comment:
Can you please split these `WalletConfig` assignments across multiple lines?

Reply:
done